cmd/xdpcap/filter_test.go

```diff
@@ -44,6 +44,51 @@ func TestMissingFilter(t *testing.T) {
 	}
 }
 
+func TestFilterProgramMultiBufferModeSupport(t *testing.T) {
```

Comment:
We use "frags" everywhere else, can we use it here too (instead of "MutliBufferModer")?

Reply:
I modified this test anyway to check that a program can be attached in both linear and multi buffer mode and I changed the name accordingly